client/src/components/UserCard.tsx

```diff
@@ -74,12 +78,15 @@ export default function UserCard(props: Props) {
               />
             )}
           </div>
+          {numOfItems === 0 && (
+            <Button large inline className="link-button" onClick={initiateLink}>
+              Add your checking or savings account
+            </Button>
+          )}
           {(props.removeButton || (props.linkButton && numOfItems === 0)) && (
             <div className="user-card__button">
               {token != null && token.length > 0 && props.linkButton && (
-                <LinkButton userId={props.userId} token={token} itemId={null}>
-                  Add your checking or savings account
-                </LinkButton>
+                <LinkButton userId={props.userId} token={token} itemId={null} />
```

Comment:
Is there supposed to be text in this button?

Reply:
no, no text in this button...I think I will do away with it being a button...

Reply:
no, no text here. I think I will make this not a button any longer